soccer/gameplay/skills/capture.py

```diff
@@ -99,7 +99,8 @@ def on_enter_course_approach(self):
 
     def execute_course_approach(self):
         # don't hit the ball on accident
-        self.robot.set_avoid_ball_radius(Capture.CourseApproachAvoidBall)
+        #self.robot.set_avoid_ball_radius(Capture.CourseApproachAvoidBall)
+        self.robot.disable_avoid_ball()
```

Comment:
I think we probably want to still avoid the ball here.  The reason it's there is because we've had problems bumping into the ball during the course approach state.  Keep in mind sim behavior may differ from reality a good bit here.